system/Debug/Exceptions.php

```diff
@@ -541,9 +541,12 @@ public static function highlightFile(string $file, int $lineNumber, int $lines =
             return false;
         }
 
+        $version = explode('.', PHP_VERSION);
+        $php_version = (int)$version[0] * 10000 + (int)$version[1] * 100 + (int)$version[2];
```

Comment:
This function is deprecated, and no longer used.
Do you really need to fix this function?
If so, use `PHP_VERSION_ID` instead. See `BaseExceptionHandler::highlightFile()`